### ISSUE TITLE
Unexplained crashes

### ISSUE THREAD

**Issue by BobFred7** [pandorabox-io/pandorabox.io#469]
There have been several crashes, including the most recent one at the time of this writing, that have an associated file in crashlogs but the file is blank
Why did this happen?
Are there relevant error messages in the main logs that just weren't copied to crashlogs, or were there crashes with no error messages?
Can this be fixed?

**Comment by S-S-X** [comment]
> associated file in crashlogs but the file is blank

Public crashlogs are filtered logs and wont show everything. It could also be hard crash in engine, those can leave normal server logs completely useless.

> Can this be fixed?

Problems can be fixed but as long as we don't know actual reason we also probably can't easily reproduce or debug it efficiently.

> were there crashes with no error messages?

There can be crashes without error messages, however that does not mean that root causes would be impossible to trace. It just requires some manual work.

Basically all this means that we don't know yet and @BuckarooBanzay probably has to be first to get hands dirty with this one.

**Comment by BuckarooBanzay** [comment]
> Basically all this means that we don't know yet and @BuckarooBanzay probably has to be first to get hands dirty with this one.

yay.. but my hands are already dirty 😄 

The current issue: #446
Upstream (engine) issue: https://github.com/minetest/minetest/issues/9497

The logs are empty because the process just stops (or gets killed because it violates the memory boundaries/attributes?), the only infos are in the kernel-logs:
```
[16530902.358513] Server[12586]: segfault at fffffffffffffff8 ip 00007f73da27a5d6 sp 00007f73d1ff98d0 error 5 in libluajit-5.1.so.2.1.0[7f73da26a000+76000]

[16602295.920728] Server[13855]: segfault at fffffffd41bd7338 ip fffffffd41bd7338 sp 00007f17467faad8 error 15

# most recent one:
[16615711.120827] Server[10329]: segfault at fffffffffffffff8 ip 00007fa8671ca5d6 sp 00007fa856ffb8d0 error 5 in libluajit-5.1.so.2.1.0[7fa8671ba000+76000]
```

The last coredump i debugged from the segfault contained very little information, only some pointers, i'll take a look at the newest core-dumps the next few days...

Closing this, feel free to comment in #446 if you have some ideas/suggestions 👍
